t/t2080-parallel-checkout-attributes.sh

```diff
@@ -0,0 +1,375 @@
+#!/bin/sh
+
+test_description='Parallel Checkout (Attributes)'
+
+. ./test-lib.sh
+
+TEST_ROOT="$(pwd)"
+
+# Create some raw data files with well-known properties.
+# These will not be under version control, but rather be used
+# to create test cases later.
+
+setup () {
```

Comment:
I was thinking that the setup would be per test case rather than cumulative like most of the Txxxx cases tend to be, so I could call it from each perf test, but i haven't needed it so far.